conda-recipes-compucell3d/meta.yaml

```diff
@@ -23,8 +23,8 @@ build:
 requirements:
```

Comment:
Not sure about including this second recipe here. On Linux, conda build tries to chain together building both recipes, and then fails due to missing dependencies for this metapackage. Obviously, this then makes it impossible to build all packages in one automated workflow, since building one of these recipes automatically triggers building the other. I don't see this behavior on Windows, but also haven't found any way to disable it on Linux. 

Does it make more sense to have a separate repo just for storing this recipe? Something like a repo for a simple conda-forge recipe, maybe. Or maybe we could just post the metapackage on conda-forge? Let's discuss.

Reply:
We can definetely introduce separate repository with compucell3d meta recipe. I added it here as a placeholder . BTW they way I am building package I do:

`cd conda-recipes`
`conda build .`

this way you are forcing conda to use recipe from current dir. But I have not tested it on Linux. Does conda beahve differently on Linux?

Reply:
This is the same procedure that I use both on Windows and Linux, so maybe it's a conda issue.